Northstar.CustomServers/mod/scripts/vscripts/gamemodes/_gamemode_fd.nut

```diff
@@ -83,12 +83,12 @@ void function GamemodeFD_Init()
 	AddSpawnCallback( "player", FD_PlayerRespawnCallback )
 	AddSpawnCallback("npc_turret_sentry", AddTurretSentry )
 	//death Callbacks
-	AddCallback_OnNPCKilled(OnNpcDeath)
-	AddCallback_OnPlayerKilled(GamemodeFD_OnPlayerKilled)
+	AddCallback_OnNPCKilled( OnNpcDeath )
+	AddCallback_OnPlayerKilled( GamemodeFD_OnPlayerKilled )
 	
 	//Command Callbacks
-	AddClientCommandCallback("FD_ToggleReady",ClientCommandCallbackToggleReady)
-	AddClientCommandCallback("FD_UseHarvesterShieldBoost",useShieldBoost)
+	AddClientCommandCallback( "FD_ToggleReady", ClientCommandCallbackToggleReady )
+	AddClientCommandCallback( "FD_UseHarvesterShieldBoost", useShieldBoost )
 
 	//shop Callback
 	SetBoostPurchaseCallback(FD_BoostPurchaseCallback)
```

Comment:
```suggestion
	SetBoostPurchaseCallback( FD_BoostPurchaseCallback )
```

Reply:
Line 95 also needs spacing but github doesn't allowing reviewing on code that isn't near the modified diff.